apps/console/src/features/applications/components/forms/advanced-configurations-form.tsx

```diff
@@ -306,7 +303,7 @@ export const AdvancedConfigurationsForm: FunctionComponent<AdvancedConfiguration
                 data-componentid={ `${ testId }-enable-authorization-checkbox` }
                 hidden={
                     !applicationConfig.advancedConfigurations.showEnableAuthorization
-                    || !UIConfig?.legacyMode?.applicationXacmlAuthorization }
+                    || !window[ "AppUtils" ].getConfig().ui.isXacmlConnectorEnabled }
```

Comment:
Shall we get this value through redux store or useUIConfig hook?

Reply:
fixed